### PR TITLE
[BRANDBOOK][SWE-1379] Created Input #done

### PR DESCRIPTION
**Pull request by Anhelina4**

### CODE REVIEW

**Comment by Anhelina4** [comment]
image

**Review by aleksandrgolyk — APPROVED** [review]

File: src/app/components/elements/Input/Input.styled.js [hunk 1]
```diff
@@ -2,10 +2,42 @@ import styled from 'styled-components'
 
 export const InputWrapper = styled.div``
```
Comment (olenashevchuk) [inline src/app/components/elements/Input/Input.styled.js:3]: delete this useless wrapper

File: src/styles/vars/input.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  /* INPUT */
+  --brandbook-module-brand-input-bg: rgba(37, 48, 55, 0.08);
+  --brandbook-module-brand-input-border-radius: 7px;
+
+  /* PLACEHOLDER */
+  /* FIXME - write mediators for colors, fontFamily, fs, lh*/
+  --brandbook-module-brand-input-placeholder-line-height: 24px;
+  --brandbook-module-brand-input-placeholder-font-family: 'GalanoGrotesque';
+  --brandbook-module-brand-input-placeholder-font-size: 16px;
+  --brandbook-module-brand-input-placeholder-color-default: rgba(
```
Comment (olenashevchuk) [inline src/styles/vars/input.css:11]: are there no vars for  colors?

File: src/styles/vars/input.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
+:root {
+  /* INPUT */
+  --brandbook-module-brand-input-bg: rgba(37, 48, 55, 0.08);
+  --brandbook-module-brand-input-border-radius: 7px;
```
Comment (olenashevchuk) [inline src/styles/vars/input.css:4]: use var --brandbook-module-brand-border-radius-default

**Review by olenashevchuk — CHANGES_REQUESTED** [review]

File: src/app/components/elements/Input/Input.js [hunk 4]
```diff
@@ -7,28 +7,32 @@ import { v4 as uuidv4 } from 'uuid'
...
- * and other props
+ * Also receives props: state,  disabled
  * @returns A React Component with passed props applied as styles
  */
 const Input = (props) => {
-  const { label, hovered, active, disabled, ...rest } = props
+  const { label, disabled, state, ...rest } = props
 
+  // [COMPUTED PROPERTIES]
   const inputId = uuidv4()
```
Comment (olenashevchuk) [inline src/app/components/elements/Input/Input.js:17]: we can use label prop like inputId

File: src/styles/vars/input.css (hunk 2 above)
Comment (Anhelina4) [reply src/styles/vars/input.css:11]: No, this colot is not in config yet

File: src/app/components/elements/Input/Input.js [hunk 5]
```diff
@@ -1,34 +1,37 @@
...
   return (
-    <InputWrapper>
+    <>
       {label && (
         <InputLabelWrapper htmlFor={inputId}>{label}</InputLabelWrapper>
       )}
-      <input {...rest} disabled={disabled} id={inputId} />
-    </InputWrapper>
+      <InputStyled
+        {...rest}
```
Comment (olenashevchuk) [inline src/app/components/elements/Input/Input.js:24]: better pass rest object the last

**Review by olenashevchuk — APPROVED** [review]

**Review by aleksandrgolyk — APPROVED** [review]

**Pull request merged**